source/extensions/filters/http/ext_authz/ext_authz.cc

```diff
@@ -7,11 +7,26 @@
 
 #include "extensions/filters/http/well_known_names.h"
 
+#include "absl/container/flat_hash_set.h"
+
 namespace Envoy {
 namespace Extensions {
 namespace HttpFilters {
 namespace ExtAuthz {
 
+namespace {
+// Asserts that the HTTP method is for a header-only request. For details
+// https://tools.ietf.org/html/rfc7231#section-4.3.
+bool isHeaderOnlyMethod(absl::string_view method) {
+  const static absl::flat_hash_set<std::string>* keys = new absl::flat_hash_set<std::string>(
```

Comment:
I roughly remember GET doesn’t have to be header only and I couldn’t find out where this is coning from in the RFC, more comments?

Reply:
No, you are right, but I think we will have problems when the stream is doing a WebSocket upgrade. I will need to look into it.
```
A payload within a GET request message has no defined semantics;
sending a payload body on a GET request might cause some existing
implementations to reject the request.
```